src/main/java/net/sf/mzmine/modules/peaklistmethods/peakpicking/adap3decompositionV2/ADAP3DecompositionV2Task.java

```diff
@@ -272,7 +285,8 @@ private Feature getFeature(@Nonnull RawDataFile file, @Nonnull BetterPeak peak)
 
     return new SimpleFeature(file, peak.getMZ(), peak.getRetTime(), peak.getIntensity(), area,
         scanNumbers, dataPoints, Feature.FeatureStatus.MANUAL, representativeScan,
-        representativeScan, Range.closed(peak.getFirstRetTime(), peak.getLastRetTime()),
+        representativeScan, new int[] {-1},
```

Comment:
Maybe change all arrays with -1 to an empty array / or null

Reply:
yes, that would make a lot of sense now to maintain consistency.